.scalafmt.conf

```diff
@@ -1,5 +1,5 @@
-version=2.7.5
-docstrings=JavaDoc
+version=3.0.3
+docstrings.style=asterisk
```

Comment:
Asterisk is the new name for the Javadoc style, see docstring in https://scalameta.org/scalafmt/docs/configuration.html